demos-europe/demosplan-core
# dplan 4020 refactor controller

### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-4020/3-5-Als-Bearbeiterin-mochte-ich-eigene-Karten-zur-Planung-einbinden-konnen-um-Lagebeziehungen-oder-nicht-unmittelbar-erkennbare

Description
1. I created a new action in the controller: DemosPlanDocumentController::saveImportedElementsAdminAction
- This action is called in the template elements_admin_import.html.twig
- More exactly, it is called when the user clicks "save" button
- What this action does is to save the imported documents
2. That action then redirects to the existing one: DemosPlanDocumentController::elementAdminListAction
- I adjusted the existing one inspired on the work that you did on DemosPlanMapController

### How to review/test
- Case 1: Go to diplanfest  and bobsh --> verfahren/PROCEDURE_ID/verwalten/planunterlagen
- import document
- cancel import and play around with it.
- It should work as expected

Delete the checkbox if it doesn't apply/isn't necessary.
